benchmarks/entropy_adiabat/plugins/entropy_model.h

```diff
@@ -26,13 +26,56 @@
 #include <aspect/utilities.h>
 #include <aspect/simulator_access.h>
 #include <aspect/material_model/rheology/ascii_depth_profile.h>
+#include <aspect/material_model/rheology/drucker_prager.h>
 
 namespace aspect
 {
   namespace MaterialModel
   {
     using namespace dealii;
 
+    namespace internal
```

Comment:
You should be able to add `#include <aspect/material_model/steinberger.h>`  at the top of the file, then you can delete this class declaration here. This does only work if you did not change something in the implementation of the LaterialViscosityLookup class.